lib/components/atoms/Video/types/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  width?: Number,
+  height?: Number,
+  autoPlay?: Boolean,
+  controls: Boolean,
+  loop?: Boolean,
+  muted?: Boolean,
+  poster?: String,
+  src: String,
+  type: String,
+  children?: Node,
+  className?: String,
+  controlsList?: String,
+  defaultCaption: Object,
```

Comment:
shouldn't defaultCaption be also optional?

Reply:
yes, done